src/main/java/net/neoforged/neoforge/common/extensions/ValueInputExtension.java

```diff
@@ -25,4 +26,15 @@ default Set<String> keySet() {
         //noinspection deprecation
         return self().read(MapCodec.assumeMapUnsafe(CompoundTag.CODEC)).orElseThrow().keySet();
     }
+
+    /**
+     * Reads the {@code child} object from the given {@code key}.
+     * The object will only read the child <strong>ONLY</strong> if it's present.
```

Comment:
```suggestion
     * The object will read the child <strong>ONLY</strong> if it's present.
```
 My brain missed the double only, apologies.